modules/adding-node-iso-flags.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/nodes-nodes-adding-node-iso.adoc
+
+[id="adding-node-iso-flags{context}"]
+= Adding a node with command flags
+
+You can add a single node to your cluster by using command flags to specify configurations for the new node.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`)
+* You have an active connection to your target cluster
+* You have a kubeconfig file available
+
+.Procedure
+
+. Generate the ISO image by running the following command. The MAC address must be specified using a command flag.
++
+[source,terminal]
+----
+$ oc adm node-image create --kubeconfig='<path_to_kube_config>' --mac-address=<mac_address>
+----
++
+--
+where:
+
+`<path_to_kube_config>`:: Specifies the path to the kubeconfig file.
+`<mac_address>`:: Specifies the MAC address of the node that is being added.
+--
++
```

Comment:
Another important point, that maybe could be added as a note or tip: the `create` command will need to fetch the ocp release image matching the target cluster version. This means that to operate correctly the command will always require a valid pull-secret (via the `--registry-config` flag). There are alternative ways to specify the pull-secret, for example by using the `REGISTRY_AUTH_FILE` env var,  so maybe it could be useful to have an example with that (or a reference to the oc docs for that flag)